tests/nested/manual/core20-early-config/task.yaml

```diff
@@ -0,0 +1,53 @@
+summary: Test that gadget config defaults are applied early on core20.
+
+systems: [ubuntu-20.04-64]
+
+prepare: |
+    ENABLE_TPM="true"
```

Comment:
Why not just set the environment in the task?

Reply:
Agree, the 3 variables should go in the env vars for the test

Reply:
Done.